proto/ethermint/evm/v1/query.proto

```diff
@@ -230,8 +227,8 @@ message EstimateGasResponse {
 message QueryTraceTxRequest {
   // msgEthereumTx for the requested transaction
   MsgEthereumTx msg = 1;
-  // transaction index
-  uint64 tx_index = 2;
+  // tx_index is not necessary anymore
+  reserved 2;
```

Comment:
is this how we deprecate fields? Would this be breaking then?

Reply:
Correct! In this case it was done based on this comment https://github.com/tharsis/ethermint/pull/1034#discussion_r846074268

Reply:
correct, this is API breaking